components/Input/index.test.jsx

```diff
@@ -0,0 +1,23 @@
+import { render, screen } from '@testing-library/react';
+
+import Input from '.';
+
+describe('Input', () => {
+  it('renders a input box', () => {
```

Comment:
```suggestion
  it('renders an input box', () => {
```

Reply:
done in https://github.com/tyrro/react-survey/commit/dbf9d1e261944188cc87b0d6f495a313bea9b890